lsr-role2collection.py

```diff
@@ -0,0 +1,443 @@
+#!/usr/bin/env python3
```

Comment:
I think this is ok - python3 should be available on all of the el7 platforms we might use